### DISCUSSION TITLE
Leaflet website is not avilable at russia

### DISCUSSION THREAD

**Discussion by WowInceptionGood** [Leaflet/Leaflet#9328] [General]
I know, that Leaflet doesn't support war or putin propoganda. But there are many devs from russia, that aren't supporting war either. And you are blocking access to website. By blocking website, you aren't blocking putin and soldiers, you are blocking developers, that are want to use Leaflet.

**Comment by jonkoops** [comment]
Define 'blocking the website'. We aren't doing anything to block the site, you are free to use the site from anywhere in the world. Russians are more than free to use Leaflet.

**Reply by mourner** [reply]
The website is actually blocked by the Russian government, officially 😂 A privilege if you ask me.

**Reply by jonkoops** [reply]
Well, I am not surprised.

**Comment by IvanSanchez** [comment]
This has been reported before as https://github.com/Leaflet/Leaflet/issues/8346 and https://github.com/Leaflet/Leaflet/issues/8515 . My comment from https://github.com/Leaflet/Leaflet/issues/8346#issuecomment-1188302270 still stands.
